src/components/Tooltip/Tooltip.stories.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../scss/obelisco.scss';
+
+.tooltip-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 5rem 0;
+
+  .tooltip-row {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    gap: 16px;
+    margin-bottom: 1rem;
+
+    &:has(.tooltip-column) {
+      flex-direction: column;
+    }
+
+    .tooltip-column {
+      width: 100%;
+      display: flex;
+      justify-content: center;
+      gap: 16px;
+    }
+  }
+}
+
+@include media-breakpoint-down(md) {
```

Comment:
Sacaría el import de Obelisco y usaría
@media screen and (max-width: 975.98px) {
 // el código
}